packages/menu/src/types.ts

```diff
@@ -66,6 +66,8 @@ export interface IUseMenuProps<T = HTMLButtonElement, M = HTMLElement> {
   focusedValue?: string | null;
   /** Determines focused value on menu initialization */
   defaultFocusedValue?: string;
+  /** Returns focus to the trigger when the menu is collapsed */
```

Comment:
[nit] Maybe this can align with our description on the modal container prop:

```
Returns keyboard focus to the element that triggered the modal
```

Reply:
+1 – `Returns keyboard focus to the element that triggered the menu`